integration/tests/cook/util.py

```diff
@@ -1602,6 +1602,30 @@ def are_pools_enabled():
     return len(active_pools(cook_url)[0]) > 1
 
 
+def are_gpus_enabled():
+    """Returns true if GPU jobs can be submitted to this cluster."""
+    cook_url = retrieve_cook_url()
+    settings_dict = settings(cook_url)
+    return settings_dict['mesos-gpu-enabled']
```

Comment:
The key checked 'mesos-gpu-enabled' seems inconsistent with 'are-gpus-enabled'? Should this be are-mesos-gpus-enabled? (Or if this is correct, can you put in a comment indicating such and explain?)

Reply:
`mesos-gpu-enabled` was a poorly named variable. It really just means if gpus are enabled in the cluster